src/utils.py

```diff
@@ -0,0 +1,62 @@
+from os import environ
+import sys
+
+import requests
+
+
+AUTH = None
+AUTH_API_ENDP = environ['AUTH_API_ENDP']
+JWT_DISABLED = environ['JWT_DISABLED'].lower() == 'true'
+URI_API_ENDP = environ['URI_API_ENDP']
+URI_API_USER = environ['URI_API_USER']
+URI_API_PASS = environ['URI_API_PASS']
+URI_SCHEME = environ['URI_SCHEME']
+URI_STRICT = environ['URI_STRICT']
+
+
+def get_token(url, email, passwd):
+    """Fetch token from tokens API."""
+    credentials = {'email': email, 'password': passwd}
+    res = requests.post(url, json=credentials)
+
+    try:
+        assert res.status_code == 200
+    except AssertionError:
+        raise ValueError(res.content.decode('utf-8'))
+
+    return res.json()['data'][0]['token']
+
+
+def doi_to_id(doi, timestamp):
+    """Query translator to convert book DOI to specified schema."""
+    params = {
+        'uri': doi,
+        'filter': f'uri_scheme:{URI_SCHEME}',
+        'strict': URI_STRICT
+    }
+
+    headers = {'Authorization': AUTH} if AUTH else {}
+
+    response = requests.get(URI_API_ENDP, params=params, headers=headers)
+
+    try:
+        assert response.status_code == 200
+    except AssertionError:
+        r = response.json()
+        print(
+            f"{r['message']}: {r['parameters']['uri']} ({timestamp})",
+            file=sys.stderr
+        )
+        return []
+
+    return response.json()['data']
```

Comment:
Will `response.json()` _always_ contain `data`?

Reply:
From what I understand, yes. Every `200` response from the Translator API will be in the form: 
`{'status': 'ok', 'code': 200, 'count': count, 'data': data}`

Reply:
Yes, every 200 response contains data